tools/bin/goreleaser_utils

```diff
@@ -13,7 +13,38 @@ before_hook() {
   install_local_plugins
   install_remote_plugins
   mkdir -p "$lib_path/plugins"
-  cp "$(go env GOPATH)"/bin/chainlink* "$lib_path/plugins"
+
+  # Retrieve GOPATH
+  GOPATH=$(go env GOPATH)
```

Comment:
Will this continue to work forever? Or is there a new/better way to do it? e.g. just fetching `GOBIN` directly in this case?

Reply:
Oh, I guess nbd - the env will stay: 
> Is the GOPATH variable being removed?
No. The GOPATH variable (set in the environment or by go env -w) is not being removed. It will still be used to determine the default binary install location, module cache location, and checksum database cache location, as mentioned at the top of this page.

https://go.dev/wiki/GOPATH#faq

Reply:
when i was testing this out, `GOBIN` was undefined on my machine, but GOPATH has always existed when i tried it across diff installs